docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:3.5
+LABEL maintainer Jan Koppe <post@jankoppe.de>
+
+ARG PUMPIO__GUID=1337
+ARG PUMPIO__UID=1337
+
+ENV PUMP_LOCATION="/usr/local/pumpio"
+ENV PUMP_GIT_TAG="v4.0.0-beta.5"
+ENV PUMP_GIT_REPO="https://github.com/pump-io/pump.io.git"
+
+RUN apk add --no-cache graphicsmagick openssl nodejs python make g++ git \
+     && mkdir -p "${PUMP_LOCATION}" \
+     && git clone --recursive --depth 1 --branch "${PUMP_GIT_TAG}" "${PUMP_GIT_REPO}" "${PUMP_LOCATION}" \
```

Comment:
Yes, this is shell. In this case the curly braces can be left out, but it is common good practice to include them to avoid accidental misinterpretation of the variable name. (e.g. (`$PUMP_GIT_TAGsomeotherthing`)

Yepp, building from local source is a good idea. I tend to use that git clone style because then the whole Dockerfile itself is neatly portable. :)